Task3/EEL_task_3/src/main/java/com/epam/esm/giftcertificate/controller/GiftCertificateController.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.esm.giftcertificate.controller;
+
+
+import com.epam.esm.giftcertificate.model.GiftCertificate;
+import com.epam.esm.giftcertificate.service.GiftCertificateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping(value = "/certificate", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
RestController annotation specify json type for produces and comsumes, so you can remove these values